crates/emmylua_ls/src/handlers/initialized/client_config/vscode_config.rs

```diff
@@ -17,6 +15,8 @@ pub async fn get_client_config_vscode(
     context: &ServerContextSnapshot,
     config: &mut ClientConfig,
 ) -> Option<()> {
+    get_client_config_default(context, config, None).await;
```

Comment:
Why request configuration? The existing configuration in EmmyLua is unrelated to the language server itself; it’s mainly for color settings.

Reply:
Manual maintenance is very difficult. LuaLS generates its package.json, so if this is to be implemented, I hope it can be generated automatically.

Reply:
With schema export already present, this shouldn't be hard to implement. I'll add it when sending PR to the VSCode extension.